packages/compiler/core/type-utils.ts

```diff
@@ -75,3 +79,27 @@ export function isGlobalNamespace(
 ): namespace is Namespace & { name: ""; namespace: undefined } {
   return program.getGlobalNamespaceType() === namespace;
 }
+
+/**
+ * Check if the given type is declared in the specified namespace or, optionally, its child namespaces.
+ * @param type Type
+ * @param namespace Namespace
+ * @returns {boolean}
+ */
+export function isDeclaredInNamespace(
+  type: Model | Operation | Interface | Namespace | Enum,
```

Comment:
I think for this function to work in all operation you should look if an operation is in an interface as it would not have the namespace assigned(i think)

Reply:
would add a test to the test case below if it jsut work at least